ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -135,6 +139,27 @@ class CommandBarButton: UIButton {
                         dark: Colors.primary(for: window))
     }
 
+    private var accentImageView: UIImageView?
+
+    private func updateAccentImage(accentImage: UIImage?) {
```

Comment:
minor: to be more swift-y, would recommend either `updateAccentImage(_ accentImage: UIImage?)` or `update(accentImage: UIImage?)`. So callers don't have to duplicate `accentImage`